internal/asm/arm64/impl_6_test.go

```diff
@@ -2,13 +2,42 @@ package arm64
 
 import (
 	"encoding/hex"
+	"fmt"
 	"testing"
 
 	"github.com/tetratelabs/wazero/internal/asm"
 	"github.com/tetratelabs/wazero/internal/testing/require"
 )
 
 func TestAssemblerImpl_EncodeRelativeJump(t *testing.T) {
+	t.Run("offset limits", func(t *testing.T) {
```

Comment:
I couldn't come up with a way to test these in the existing tests because it would mean having many megabytes of `expHex` making the test unwieldy.

For similar reasons, I couldn't come up with a way to test the negative cases here, since I think it requires a backwards jump, which then ends up again needing megabytes of code due to this line

https://github.com/tetratelabs/wazero/blob/main/internal/asm/arm64/impl.go#L756

Let me know if this is OK or any ideas for improving